modules/ocf_mirrors/manifests/archlinux.pp

```diff
@@ -9,6 +9,12 @@
       recurse => true;
   }
 
+  ocf_mirrors::monitoring { 'archlinux':
+    type          => 'ts',
+    upstream_host => 'mirrors.kernel.org',
```

Comment:
I wish there was a more official upstream here, but I can't find one (I think they whitelist access to the direct mirror, and might only allow rsync), so I think this is ok.